src/ObjectUtils/ElfFile.cpp

```diff
@@ -416,6 +418,76 @@ ErrorMessageOr<ModuleSymbols> ElfFileImpl<ElfT>::LoadSymbolsFromDynsym() {
   return module_symbols;
 }
 
+template <typename ElfT>
+ErrorMessageOr<orbit_grpc_protos::ModuleSymbols>
+ElfFileImpl<ElfT>::LoadEhOrDebugFrameEntriesAsSymbols() {
+  const std::unique_ptr<llvm::DWARFContext> dwarf_context =
+      llvm::DWARFContext::create(*object_file_);
+  constexpr const char* kErrorMessage =
+      "Unable to load unwind info ranges from the .debug_frame or the .eh_frame section.";
+  if (dwarf_context == nullptr) {
+    return ErrorMessage{kErrorMessage};
+  }
+
+  const llvm::DWARFDebugFrame* debug_or_eh_frame = nullptr;
+  bool is_eh_frame = false;
+
+  if (llvm::Expected<const llvm::DWARFDebugFrame*> debug_frame = dwarf_context->getDebugFrame();
```

Comment:
I think this deserves a comment (similar to the one in libunwindstack) that we prioritize debug_frame over eh_frame.